[0m03:20:32.689748 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000288CD59D0D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000288CD59E810>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000288CD59EB10>]}


============================== 03:20:32.717692 | c5ad7086-5a5f-4ed4-9dfc-a25f35ce4017 ==============================
[0m03:20:32.717692 [info ] [MainThread]: Running with dbt=1.10.1
[0m03:20:32.717692 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\DELL\\.dbt', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'introspect': 'True', 'invocation_command': 'dbt init dbt', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m03:21:24.942005 [debug] [MainThread]: Starter project path: C:\Users\DELL\Documents\Github\windows_venv\Lib\site-packages\dbt\include\starter_project
[0m03:21:25.002148 [info ] [MainThread]: 
Your new dbt project "dbt_" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m03:21:25.004159 [info ] [MainThread]: Setting up your profile.
[0m03:24:47.896466 [info ] [MainThread]: Profile dbt_ written to C:\Users\DELL\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m03:24:48.085030 [debug] [MainThread]: Command `dbt init` succeeded at 03:24:48.074544 after 255.57 seconds
[0m03:24:48.107742 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000288C7746050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000288CD6185D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000288CD61A750>]}
[0m03:24:48.140182 [debug] [MainThread]: Flushing usage events
[0m03:24:49.640218 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m03:25:08.307792 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000026141D9A8D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000026141D9A990>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000026141D9A690>]}


============================== 03:25:08.320880 | 47ffbfd1-2f08-458c-9f14-4102829cb3b3 ==============================
[0m03:25:08.320880 [info ] [MainThread]: Running with dbt=1.10.1
[0m03:25:08.322891 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\DELL\\.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m03:25:08.419139 [info ] [MainThread]: dbt version: 1.10.1
[0m03:25:08.419139 [info ] [MainThread]: python version: 3.11.9
[0m03:25:08.419139 [info ] [MainThread]: python path: C:\Users\DELL\Documents\Github\windows_venv\Scripts\python.exe
[0m03:25:08.419139 [info ] [MainThread]: os info: Windows-10-10.0.26100-SP0
[0m03:25:10.034907 [info ] [MainThread]: Using profiles dir at C:\Users\DELL\.dbt
[0m03:25:10.036921 [info ] [MainThread]: Using profiles.yml file at C:\Users\DELL\.dbt\profiles.yml
[0m03:25:10.041036 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\DELL\Documents\Github\Real-Time-Shipment-Tracking\dbt_project.yml
[0m03:25:10.045817 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m03:25:10.054852 [debug] [MainThread]: Command `dbt debug` failed at 03:25:10.053855 after 1.90 seconds
[0m03:25:10.056679 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000026141DF9A10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000026143B705D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002613CE45F10>]}
[0m03:25:10.060383 [debug] [MainThread]: Flushing usage events
[0m03:25:11.332807 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m03:27:03.648053 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000276DF09B110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000276DF09CF50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000276DF09EB90>]}


============================== 03:27:03.665437 | d1f8d595-13fb-4fdc-be5e-ad1f95820344 ==============================
[0m03:27:03.665437 [info ] [MainThread]: Running with dbt=1.10.1
[0m03:27:03.667432 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\DELL\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m03:27:03.766063 [info ] [MainThread]: dbt version: 1.10.1
[0m03:27:03.766063 [info ] [MainThread]: python version: 3.11.9
[0m03:27:03.776858 [info ] [MainThread]: python path: C:\Users\DELL\Documents\Github\windows_venv\Scripts\python.exe
[0m03:27:03.783699 [info ] [MainThread]: os info: Windows-10-10.0.26100-SP0
[0m03:27:05.584493 [info ] [MainThread]: Using profiles dir at C:\Users\DELL\.dbt
[0m03:27:05.600246 [info ] [MainThread]: Using profiles.yml file at C:\Users\DELL\.dbt\profiles.yml
[0m03:27:05.600246 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\DELL\Documents\Github\Real-Time-Shipment-Tracking\dbt_project.yml
[0m03:27:05.600246 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m03:27:05.608068 [debug] [MainThread]: Command `dbt debug` failed at 03:27:05.606966 after 2.20 seconds
[0m03:27:05.609067 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000276D9135F10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000276DFE70550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000276DF0FB990>]}
[0m03:27:05.610922 [debug] [MainThread]: Flushing usage events
[0m03:27:06.662682 [debug] [MainThread]: An error was encountered while trying to flush usage events
